pkg/kopia/command/storage/secret_utils.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+type LocType string
+
+const (
+	LocTypeS3        LocType = "s3"
+	LocTypeGCS       LocType = "gcs"
+	LocTypeAzure     LocType = "azure"
+	LocTypeFilestore LocType = "filestore"
```

Comment:
Let's not export the consts unless required. I can see it's being used in `storage` pkg only.

Reply:
Since I broke a PR into multiple PRs, we cannot see the variables being used. We would need to export them eventually.